### PR TITLE
[WEB-162] Disable Profile Modal

### PR DESCRIPTION
**Pull request by 314numberpi**

### CODE REVIEW

File: client/components/DisableProfileModal.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            onChange={this._handleChange}
+            placeholder="Password"
+            value={password}
+            required />
+        </fieldset>
+
+        <ModalFooter>
+          <div className="secondary" />
+          <div className="primary">
+            <button
```
Comment (UncleClapton) [inline client/components/DisableProfileModal.js]: to be extra annoying, we should add another confirmation button :D

**Review by UncleClapton — CHANGES_REQUESTED** [review]

File: client/components/DisableProfileModal.js (hunk 1 above)
Comment (314numberpi) [reply client/components/DisableProfileModal.js]: I was thinking of it :P A job for another day.

**Comment by UncleClapton** [comment]
PR blocked by work required on applayout.

**Comment by UncleClapton** [comment]
This PR is no longer blocked.

**Review by UncleClapton — APPROVED** [review]

**Comment by UncleClapton** [comment]
LGTM!

**Review by UncleClapton — CHANGES_REQUESTED** [review]
ACTUALLY. could you add your changes to the changelog under unreleased? We need to start tracking v3 changes.

**Review by UncleClapton — APPROVED** [review]

**Pull request merged**